pwndbg/gdblib/disasm/instruction.py

```diff
@@ -232,6 +233,21 @@ def __init__(self, cs_insn: CsInsn | None) -> None:
         FALSE if the instruction has a conditional action, and we know it is not taken.
         """
 
+        self.declare_conditional: bool | None = None
+        """
```

Comment:
New field necessary for handling Arm/Arch branch instruction conditionality as they relate to how we unroll jumps in the disasm view.